src/commands/logout.ts

```diff
@@ -83,6 +83,9 @@ async function logout(_options?: LogoutOptions): Promise<void> {
     log('Removing Auth0 tokens from keychain');
     cliOutput(`\n${chalk.blue('i')} Clearing authentication data...\n`);
 
+    log('Revoke refresh token if present');
+    revokeRefreshToken();
```

Comment:
This call to isn't `await`ed, which means keychain clearing might happen before revocation completes.

Reply:
Would be useful to make `revokeRefreshToken` return a `boolean` for success/failure so callers can handle errors more effectively. Right now it just swallows the exception (which is by design I think).

Reply:
> which means keychain clearing might happen before revocation completes.

I don't think that'd matter unless we need retries.